docs/pages/management/export-audit-events/splunk.mdx

```diff
@@ -0,0 +1,791 @@
+
+  Run the following commands on your Universal Forwarder host:
+
+  ```code 
+  $ git clone https://github.com/gravitational/teleport-plugins.git --depth 1
+  $ cd teleport-plugins/event-handler
+  $ go build
+  ```
+
+  The resulting executable will have the name `event-handler`. To follow the
+  rest of this guide, rename this file to `teleport-event-handler` and move it
+  into your `PATH`.
+
+  </TabItem>
+</Tabs>
+
+### Generate a starter config file
+
+On the host where you are running the Universal Forwarder and the Event Handler
+plugin, execute the following command to generate a sample configuration that we
+will edit later. Replace `mytenant.teleport.sh` with the DNS name of your
+Teleport Proxy Service or Teleport Cloud tenant:
+
+```code
+$ teleport-event-handler configure . mytenant.teleport.sh
+```
+
+The `teleport-event-handler configure` command generates several files that you
+will refer to when generating credentials for the Event Handler plugin and
+configuring the Universal Forwarder:
+
+|File(s)|Purpose|
+|---|---|
+| `ca.crt` | Self-signed CA certificate that the Event Handler plugin and the Universal Forwarder will use to validate one another's TLS certificates.|
+| `server.crt` and `server.key`| TLS certificate and key for the Universal Forwarder.|
+| `client.crt` and `client.key`| TLS certificate and key for the Event Handler plugin.|
+| `teleport-event-handler-role.yaml`| `user` and `role` resource definitions for the Event Handler plugin.|
+| `fluent.conf`| Fluentd configuration file. We will use parts of this to configure the Universal Forwarder.|
+| `teleport-event-handler.toml` | Starter configuration file for the Event Handler plugin.|
+| `ca.key` | Not used in this setup. |
+
+### Define RBAC resources
+
+The `teleport-event-handler configure` command generated a file called
+`teleport-event-handler-role.yaml`. This file defines a `teleport-event-handler`
+role and a user with read-only access to the `event` API:
+
+```yaml
+kind: role
+metadata:
+  name: teleport-event-handler
+spec:
+  allow:
+    rules:
+      - resources: ['event', 'session']
+        verbs: ['list','read']
+version: v5
+---
+kind: user
+metadata:
+  name: teleport-event-handler
+spec:
+  roles: ['teleport-event-handler']
+version: v2
+```
+
+Move this file to your workstation (or recreate it by pasting the snippet above)
+and use `tctl` on your workstation to create the role and the user:
+
+```code
+$ tctl create -f teleport-event-handler-role.yaml
+# user "teleport-event-handler" has been created
+# role 'teleport-event-handler' has been created
+```
+
+### Enable impersonation of the Event Handler plugin user
+
+In order for the Event Handler plugin to forward events from your Teleport
+cluster, it needs signed credentials from the cluster's certificate authority.
+The `teleport-event-handler` user cannot request this itself, and requires
+another user to **impersonate** this account in order to request credentials.
```

Comment:
In how-to guides, I've been advising against putting links in steps, since that might delay the time it takes to reach the goal of the guide. I've made it clearer that we'll describe impersonation in this section, and added a link to the impersonation guide in the "Next steps" section.